net/DescriptionViewController.m

```diff
@@ -68,18 +68,49 @@ - (void) viewWillAppear:(BOOL)animated
     [self prepareUIChangeStatusElements];
     [self.tabBarController.tabBar.items objectAtIndex:1].title = @"Description";
     
-    self.issueImageView.image = nil;
-    self.issueImageView.backgroundColor = [UIColor clearColor];
-    [self.dataSorce requestImageWithName:self.currentIssue.attachments andViewControllerHandler:^(UIImage *image) {
-        dispatch_async(dispatch_get_main_queue(), ^{
-            self.issueImageView.image = image;
-        });
-    } withErrorHandler:^(NSError *error) {
-        // error
-    }];
+    
+    if (![self isActualImageView])
+    {
+        if(self.image==nil)
+        {
+            self.issueImageView.image = nil;
+            self.issueImageView.backgroundColor = [UIColor clearColor];
+        }
+        else
+        {
+            self.issueImageView.image=self.image;
+            self.actualImageView = YES;
+        }
+    }
+//    [self.dataSorce requestImageWithName:self.currentIssue.attachments andViewControllerHandler:^(UIImage *image) {
+//        dispatch_async(dispatch_get_main_queue(), ^{
+//            self.issueImageView.image = image;
+//        });
+//    } withErrorHandler:^(NSError *error) {
+//        // error
+//    }];
 
 }
 
+//-(void)viewDidAppear:(BOOL)animated
+//{
+//
+//
+//}
+
+-(void)setImage:(UIImage *)image
+{
+    _image = image;
+
+    if(self.issueImageView !=nil)
```

Comment:
It can't be nil. If issue hasn't image, model will download default image. But maybe it would be rather better to have own default images for avatar and for issue. I'll add task.